### DISCUSSION TITLE
TV Version for firesticks or android

### DISCUSSION THREAD

**Discussion by NetSven** [jellyflix-app/jellyflix#105] [Ideas]
Hello, this is for Movies and series and we would like watch on tv. Why don't have television version?

**Comment by jdk-21** [comment]
Hi, you can technically run the app on Android based TVs/streaming sticks, but your experience will be awful. Why? Because TVs have terrible performance compared to smartphones and they have completely different input methods. 
Can we implement features, that accommodate those constraints? Yes, but it takes time and developers that implement the changes. 
We already have an issue that tracks TV compatibility. 
Please be patient and in the meantime you can use one of the other client out there.

**Reply by NetSven** [reply]
Thanks, sorry bout that I'didnt know that. I just noticed I can't log in with android version on tv.

**Comment by jdk-21** [comment]
Closing in favor of #183
